data/building-hours/9-piper-center.yaml

```diff
@@ -0,0 +1,17 @@
+name: Piper Center
+category: Offices
+
+schedule:
+  - title: Hours
+    hours:
+      - {days: [Mo, Tu, We, Th, Fr], from: '8:00am', to: '5:00pm'}
+      - {days: [Tu, We], from: '7:00pm', to '9:00pm'}
```

Comment:
The Travis failure: 
```json
building-hours/9-piper-center.yaml is invalid
{
  "keyword": "additionalProperties",
  "dataPath": ".schedule[0].hours[1]",
  "schemaPath": "#/additionalProperties",
  "params": {
    "additionalProperty": "to '9:00pm'"
  },
  "message": "should NOT have additional properties"
}
```

Looking at the file, I think you missed the colon between "to" and the time.